### ISSUE TITLE
autoplay feature not working with MPV player 

### ISSUE THREAD

**Issue by cbedroid** [cbedroid/pydatpiff#2]
In **pydatpiff.media**, autoplay is set to True, but is not playing next track.
It works when using VLC player instead of MPV player. 

```python
mix = pydatpiff.Mixtapes()
media = pydatpiff.Media(mix,player='mpv')
media.setMedia(1)
media.play(1)
media.autoplay = True
```
Once the current track  finishes, the next track will not play automatically,
It just gives me a blank screen 

- Device/OS:  Ubuntu 16.04.6 LTS
- Pydatpiff version: 1.2.2
- Python version: 3.8

**Comment by cbedroid** [comment]
Working on issue #2

**Comment by cbedroid** [comment]
Closed issue resolve in  PR #4
